backend/src/main/kotlin/de/instatonne/backend/apis/PostsApiController.kt

```diff
@@ -42,7 +42,7 @@ class PostsApiController(
         val user = userService.getCurrentUser() ?: return ResponseEntity.status(HttpStatus.UNAUTHORIZED).build()
         val post = postService.createPostForUser(user)
 
-        val filePath = "/var/tmp/instatonne/${post.id}.jpg"
+        val filePath = "C:\\Users\\A14AEC9\\Documents\\DHBW\\Fächer\\Semester 4\\Web Engineering II\\tmp\\${post.id}.jpg"
```

Comment:
Also das sollte zumindest nicht in der PR sein, am besten finden wir eine Lösung, die auf jedem Betriebssystem funktioniert, aber wir sollten nicht jedes mal das mit committen, wie es gerade für uns läuft

Reply:
Oh sorry, hab ich ganz vergessen